packages/markdown-medic/src/rules/index.ts

```
@@ -0,0 +1,62 @@
+import type { CheckOptions, CheckResult } from '../check.js';
+import type { FixOptions } from '../fix.js';
+import { checkConsistentTerminology, fixConsistentTerminology } from './consistent-terminology.js';
+import { checkCodeBlockLanguage } from './code-block-language.js';
+import { checkFrontmatterRequired } from './frontmatter-required.js';
+import { checkTocRequired } from './toc-required.js';
+import { checkNoDeadLinks } from './no-dead-links.js';
+
+export async function applyCustomRules(
+  content: string,
+  options: CheckOptions
+): Promise<CheckResult[]> {
```

ESLint

[error] Array type using 'CheckResult[]' is forbidden. Use 'Array<CheckResult>' instead.

(@typescript-eslint/array-type)